pkg/hpke/hpke.go

```diff
@@ -0,0 +1,166 @@
+// Package hpke contains functions for working with Hybrid Public Key Encryption.
+package hpke
+
+import (
+	"crypto/rand"
+	"encoding/base64"
+	"fmt"
+
+	"github.com/cloudflare/circl/hpke"
+	"github.com/cloudflare/circl/kem"
+)
+
+var (
+	kemID  = hpke.KEM_X25519_HKDF_SHA256
```

Comment:
Why algo suite? All very sane choices, just curious.

Reply:
they were the ones recommended in https://github.com/jedisct1/go-hpke-compact